models/base_sources/aave_base_sources.yml

```diff
@@ -0,0 +1,40 @@
+version: 2
+
+sources:
+  - name: aave_v2_ethereum
+    description: "Decoded contracts for aave v2 on Ethereum"
+
+    tables:
+      - name: LendingPool_evt_ReserveDataUpdated
+        loaded_at_field: evt_block_time
+        description: "Provides the liqudity index, stable and variable borrow rates for aave v2 reserves."
+        freshness:
+          warn_after: { count: 12, period: hour }
+          error_after: { count: 24, period: hour }

```

Comment:
on the source freshness, we've noticed many decoded tables aren't actually refreshed within the 24-hour window, can you confirm this source decoded table should be? if so, we can leave, otherwise we can take out the error option

Reply:
Yes it's better if we keep the check for this abstraction